src/app/markers.ts

```diff
@@ -53,9 +58,11 @@ export function roundMarkerFromRoundFlower(
   flower: RoundFlower,
   barcodeValue: number,
   currentBeePollen: number,
-  recentFlowerInteractions: Interaction[]
+  recentFlowerInteractions: Interaction[],
+  incompatibleFlower: boolean
 ): RoundMarker {
-  const canVisit = canVisitFlower(
+  const lastVisitedIncompatible = visitingIncompatible(barcodeValue, recentFlowerInteractions);
```

Comment:
Since it's only used once, I think we can collapse the function here.

```suggestion
  const lastVisitedIncompatible = recentFlowerInteractions[0].incompatibleFlower && recentFlowerInteractions[0]?.barcodeValue === barcodeValue
```
(and remove the function below)